cmd/libs/go2idl/client-gen/test_apis/testgroup/v1/types.generated.go

```diff
@@ -159,7 +159,13 @@ func (x *TestType) CodecEncodeSelf(e *codec1978.Encoder) {
 				z.EncSendContainerState(codecSelfer_containerArrayElem1234)
 				if yyq2[2] {
 					yy10 := &x.ObjectMeta
-					yy10.CodecEncodeSelf(e)
```

Comment:
indeed.

I wonder whether we can detect – via reflection in the codecgen tool – whether the self codec implementation comes from `ObjectMeta` or not.